### PR TITLE
Update to v1.22.1

### PR DESCRIPTION
**Pull request by mcg1969**
- moved sha256 to top
- updated version
- updated protobuf dependency
- cleaned up metadata

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
We might want to remove the commented out section.
https://github.com/AnacondaRecipes/proto-plus-feedstock/blob/v1.22.1/recipe/meta.yaml#L16

**Comment by mcg1969** [comment]
@varlackc done!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/proto-plus-{{ version }}.tar.gz
-  sha256: 4ca4055f7c5c1a2239ac7a12770a76a16269f58d3f01631523c20fc81dbb14a7
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
+  script: python -m pip install . -vv --no-deps
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: I guess `{{ PYTHON }}` is better than `python`.

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (mcg1969) [reply recipe/meta.yaml:16]: I see both idioms out there, but I'm not convinced. If conda build is working properly, then `python` should resolve to the proper path.

**Pull request merged**